File: src/main/resources/output/IFC4X3_python/Entities/IfcMaterialDefinition/IfcMaterialConstituent/IfcMaterialConstituent.py
class IfcMaterialConstituent(IfcMaterialDefinition):
    def __init__(self):
        self.Name: IfcLabel = None
        self.Description: IfcText = None
        self.Material: IfcMaterial = None
        self.Fraction: IfcNormalisedRatioMeasure = None
        self.Category: IfcLabel = None

    # === INVERSE CLAUSES ===
    # ToMaterialConstituentSet : IfcMaterialConstituentSet FOR MaterialConstituents

File: src/main/resources/output/IFC4X3_java/Entities/IfcMaterialDefinition/IfcMaterialConstituent/IfcMaterialConstituent.java
public class IfcMaterialConstituent extends IfcMaterialDefinition {
    public IfcLabel Name;
    public IfcText Description;
    public IfcMaterial Material;
    public IfcNormalisedRatioMeasure Fraction;
    public IfcLabel Category;

    // === INVERSE CLAUSES ===
    // ToMaterialConstituentSet : IfcMaterialConstituentSet FOR MaterialConstituents
}
